app/main.py

```diff
@@ -1,34 +1,81 @@
 class Deck:
-    def __init__(self, row, column, is_alive=True):
-        pass
+    def __init__(self, row: int, column: int, is_alive: bool = True) -> None:
```

Comment:
It is more conventional to place each argument on a new line in the `__init__` to improve the code readability. Apply this here and in the other classes as well.